autoload/phpcd.vim

```diff
@@ -56,7 +56,7 @@ function! phpcd#CompletePHP(findstart, base) " {{{
 
 		let [current_namespace, imports] = phpcd#GetCurrentNameSpace()
 
-		if context =~? '\v^use\s*' " {{{
+		if context =~# '\v^(use$|use+\s.*)' " {{{
```

Comment:
The `use+\s.*` seems not right. The `+` means **one or more** `e`.

I think `\v^use$` is enough.

Reply:
Thanks revied.

It is for use when it is completion in the middle of input of `use`.

Reply:
**tmux**

<img width="982" alt="phpcd18031701" src="https://user-images.githubusercontent.com/14961986/37551246-9fd47e24-29df-11e8-9ae1-a6214bc5adfc.png">

Reply:
Even in the `use ...` case, the `context` will be `use`

Reply:
Thanks debug function.

## php

```
<?php
use Lvht\|
```

## :echo phpcd#GetCurrentSymbolWithContext()


```
['', 'use', '\Lvht', {'@': ''}]
```

## \v^use$ snapshot

<img width="916" alt="1" src="https://user-images.githubusercontent.com/14961986/37551485-b685d83a-29e3-11e8-899d-f90f4addb867.png">

## \v^(use$|use+\s.*) snapshot

<img width="909" alt="2" src="https://user-images.githubusercontent.com/14961986/37551489-bfbe1e94-29e3-11e8-9f76-643d971c7f22.png">

Reply:
Thank you for merge.

phpcd.vim great!